src/django/api/management/commands/post_deployment.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+
+
+class Command(BaseCommand):
+    help = ('Usage: This management command runs the Django migrations during '
+            'the deployment process. It could be expanded to include other '
+            'post-deployment tasks.')
+
+    def handle(self, *args, **options):
+        call_command('migrate')
```

Comment:
Is it fine that option "api" missed here? Like: "migrate,api" was before.
cc @mazursasha1990

Reply:
@roman-stolar It should help to run all migrations (account, admin, api, auth, authtoken, contenttypes, sessions, sites, socialaccount, waffle), not only for `api`.